src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -2341,7 +2341,7 @@ bool QgsPostgresProvider::addFeatures( QgsFeatureList &flist, Flags flags )
 
       if ( !mGeneratedValues.value( idx, QString() ).isEmpty() )
       {
-        QgsDebugMsg( QStringLiteral( "Skipping field %1 (idx %2) which is GENERATED." ).arg( fieldname, idx ) );
+        QgsLogger::warning( tr( "Skipping field %1 which is GENERATED." ).arg( fieldname ) );
```

Comment:
This will lead to warnings on every "add feature" with generated columns, correct?

Reply:
Yes, correct.

Reply:
I've removed the "idx" from the message because that shouldn't matter to the user, and developers can easily find out the field index if needed.

Reply:
I would completely remove this (or make it a debug message as before), as it currently is, the log will be spammed and there is no way out.

Reply:
Of course, this hadn't ocurred to me; I was thinking about inserting features only with the user-editable fields. Reverted to debug message in 0d1a4c5.